container_pipeline/lib/command.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+import subprocess
+
+
+def run(cmd):
+    """
+    Runs a shell command.
+
+    :param cmd: Command to run
+    :type cmd: str
+
+    :return: Command output
+    :rtype: str
+    :raises: subprocess.CalledProcessError
+    """
+    return subprocess.check_output(cmd, shell=True)
```

Comment:
If there's an exception in executing the `cmd`, does the caller need to handle the raised exception or should we have that logic here?